roles/controller_casc_from_aap/templates/current_projects_improved.j2

```diff
@@ -0,0 +1,20 @@
+---
+controller_projects:
+{% for project in projects_lookvar %}
```

Comment:
current_projects_asset_value if you want to follow the structure that i told you before.

Reply:
Followed same structure